relay-server/src/utils/dynamic_sampling.rs

```diff
@@ -12,10 +12,18 @@ use serde::{Deserialize, Deserializer, Serialize};
 
 use relay_common::{ProjectId, ProjectKey, Uuid};
 use relay_filter::GlobPatterns;
+use relay_general::protocol::{Event, EventId};
 
 use crate::actors::project::{GetCachedProjectState, GetProjectState, Project, ProjectState};
 use crate::envelope::{Envelope, ItemType};
 
+#[derive(Debug, Clone, Copy, Serialize, Deserialize, PartialEq, Eq)]
+#[serde(rename_all = "camelCase")]
+pub enum SamplingRuleCategory {
```

Comment:
Since we use the word "category" for data categories, but this will control something slightly different, would call this maybe "strategy":

```suggestion
pub enum SamplingStrategy {
```

Reply:
ok